neo/rawio/openephysbinaryrawio.py

```diff
@@ -359,7 +357,15 @@ def _get_signal_t_start(self, block_index, seg_index, stream_index):
 
     def _get_analogsignal_chunk(self, block_index, seg_index, i_start, i_stop,
                                 stream_index, channel_indexes):
+        info = self._sig_streams[block_index][seg_index][stream_index]
         sigs = self._sig_streams[block_index][seg_index][stream_index]['memmap']
+
+        # take care of SYNC channel
+        channel_names = [ch["channel_name"] for ch in info["channels"]]
+        has_sync_channel = any(["SYNC" in ch for ch in channel_names])
```

Comment:
This should not be done at every chunk bu in the parse header once for all.

Reply:
ok, I'll add a `has_sync_channel` in the `self._sig_streams[block_index][seg_index][stream_index]` dictionary